metatron-app/metatron-discovery
# fn-fix_ds_workspace

### Description

2856
1. Name duplication check is not working 
2. Schema info text is different
3. Navigation after creating workspace is wrong
4. toast is not shown because button is disabled when invalid input is exist 

2942
show item count in select box 

2877
show list items according to item count option

3006
name duplication check when edit workspace name and show toast when something went wrong

**Related Issue** : 

METATRON-2856
METATRON-2942
METATRON-2877
METATRON-3006
### How Has This Been Tested?


2856
1. Main -> workspacelist button 
2. click create new workspace button
3. check button is enabled
4. check name duplication is prevented
5. check toast is shown when wrong input is entered

2942 
1. Main page
2. check select box show item count 

2877
login using admin account
1. LNB -> Administarion ->  workspace
2. check show item list according to option
#### Need additional checks?

3006
1.Main -> workspacelist button
2.click work space user owned 
3.click more button and click edit name and description
4.check name duplication function works well


### Types of changes



### Checklist:


### Additional Context
